src/parsers/manifestjson.js

```diff
@@ -92,6 +93,7 @@ export default class ManifestJSONParser extends JSONParser {
       selfHosted = getDefaultConfigValue('self-hosted'),
```

Comment:
Here we pass the value of the `self-hosted` flag, I am wondering if we could/should do the same for `privileged`. I think it's going to be needed for #4333. WDYT?

Reply:
@willdurand sure, makes sense. I applied this diff as if and included it in the last updated on this PR (btw, thanks for the diff :heart:)